src/main/java/in/org/projecteka/hiu/consent/ConsentRepository.java

```diff
@@ -0,0 +1,36 @@
+package in.org.projecteka.hiu.consent;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import in.org.projecteka.hiu.consent.model.ConsentRequestDetails;
+import io.vertx.pgclient.PgPool;
+import io.vertx.sqlclient.Tuple;
+import lombok.SneakyThrows;
+import reactor.core.publisher.Mono;
+
+public class ConsentRepository {
+    private PgPool dbClient;
+
+    public ConsentRepository(PgPool pgPool) {
+        this.dbClient = pgPool;
+    }
+
+    @SneakyThrows
+    public Mono<Void> insertToConsentRequest(
+            String consentRequestId,
+            ConsentRequestDetails consentRequestDetails) {
```

Comment:
In my opinion, having extra suffix like ConsentRequest**Details** / **Info** are additional noise which can be avoided as much as possible.